keymanager-client/src/client.rs

```diff
@@ -94,7 +94,7 @@ impl RemoteClient {
         keys_cache_sizes: usize,
     ) -> Self {
         #[cfg(target_env = "sgx")]
-        init_trusted_policy_signers();
+        set_trusted_policy_signers(default_trusted_policy_signers()); // TODO: configurable?
```

Comment:
Yes this must be configurable otherwise you can't instantiate proper clients at all.

Why not pass the policy as an argument?

Reply:
>Why not pass the policy as an argument?

That was my question as well, hence the comment. Will change it

Reply:
done